projects/swc-starter-webpack/src/attach-event-helper.js

```diff
@@ -42,3 +42,23 @@ function attachEvents(tabName) {
         eval(eventListener3);
     }
 }
+
+function handleThemeColor(selectObject) {
+    var value = selectObject.value;
+    document.querySelector('#theme-block').setAttribute('color', value);
+    var globalDiv = document.getElementById('global');
+    if (
+        value == 'dark' ||
+        value == 'darkest' ||
+        value == 'light' ||
+        value == 'lightest'
+    ) {
+        globalDiv.style.backgroundColor =
```

Comment:
We are setting the same background color for all themes?

Reply:
It's a variable which takes appropriate color based on the theme selected, similar variable is being used in SWC storybook as well to provide coordinated background for the corresponding themes. I will change the implementation a bit applying this color variable directly over the div id#global